recipes/paratext/meta.yaml

```diff
@@ -0,0 +1,36 @@
+{% set version = "0.1.1" %}
+
+package:
+  name: paratext
+  version: {{ version }}
+
+source:
+  git_url: https://github.com/wiseio/paratext.git
+  git_rev: 2f2824eb8c8f1d060c1105531bc7ab9c44b55246
+
+build:
+  number: 1
+  skip: True  # [win or osx or py3k]
+
+requirements:
+  build:
+    - python
+    - setuptools
+    - swig
+    - pandas
+  run:
+    - python
+    - pandas
+
+test:
+  imports:
+    - paratext
```

Comment:
That would be nice indeed. At the moment the library doesn't seem to have any tests in the repo. I could certainly a very simple write one and add it to the feedstock for now. I did read somewhere that thats probably not something we want in feedstocks since it should really be in the library. I believe the library will add more tests soon that we can call in later versions of the pkg.